index.js

```diff
@@ -15,22 +15,15 @@ const bubleOpts = {
   }
 }
 
-const shouldSkip = file => contains(file, 'node_modules')
+const transformFile = function (filename, options = {}) {
+  const babelOptions = options.babel ? Object.assign({}, babelOpts, options.babel) : babelOpts
+  const bubleOptions = options.buble ? Object.assign({}, bubleOptions, options.buble) : bubleOpts
 
-const original = require.extensions['.js']
+  const { code: babelCode } = transformFileSync(filename, babelOptions)
```

Comment:
I'm actually curious about this one a bit. If the transform fails it should likely still throw the error as we do not want the transformation to continue, right?